Directory.Packages.props

```diff
@@ -82,10 +82,8 @@
     <PackageVersion Include="Proto.Persistence.SqlServer" Version="1.6.0" />
     <PackageVersion Include="Proto.Remote" Version="1.6.0" />
     <PackageVersion Include="pythonnet" Version="3.0.3" />
-    <PackageVersion Include="Quartz" Version="3.11.0" />
-    <PackageVersion Include="Quartz.Extensions.DependencyInjection" Version="3.11.0" />
-    <PackageVersion Include="Quartz.Extensions.Hosting" Version="3.11.0" />
-    <PackageVersion Include="Quartz.Serialization.Json" Version="3.11.0" />
+    <PackageVersion Include="Quartz.Extensions.Hosting" Version="3.13.0" />
+    <PackageVersion Include="Quartz.Serialization.Json" Version="3.13.0" />
```

Comment:
There's new SystemTextJson package for serialization which should preferred over the Newtonsoft based, I'm just unsure how much of a breaking change that would be

Reply:
I'll do some testing to see if things break when switching. Thanks for pointing out.